### PR TITLE
chore: add hub links

### PR DESCRIPTION
**Pull request by rkelson**
remove title and abstract download

### CODE REVIEW

File: data/geoscience/alluvial-fans/index.html [hunk 1]
```diff
@@ -7,14 +7,11 @@
...
 AlluvialFans:
-  downloads:
-    - url: https://drive.google.com/a/utah.gov/uc?id=1Ngbke3E0JMF5YZb_XsfSGJFitsIaWt0A&export=download
-      label: 'Alluvial Fans: File Geodatabase'
-    - url: https://drive.google.com/a/utah.gov/uc?id=1LpFkYmn-RIL1YXW-Ckn9PmvMj0_R9EoD&export=download
-      label: 'Alluvial Fans: Shapefile'
+  hub:
+    name: Utah Alluvial Fans
+    item_id: 9bdcd03de4b64cffae7e10b58e07757e
     - url: https://drive.google.com/a/utah.gov/uc?id=1VbWmRBQrDsyYjvfgZYWlox618zI718zi&export=download
```
Comment (steveoh) [inline data/geoscience/alluvial-fans/index.html:15]: these `- url and label things need to be removed`

**Comment by rkelson** [comment]
I forgot to move the drive links to the related resource area. However it looks like I messed up the downloads completely. What went wrong.

**Comment by steveoh** [comment]
> I forgot to move the drive links to the related resource area. However it looks like I messed up the downloads completely. What went wrong.

i think once you remove them the downloads will work

**Comment by rkelson** [comment]
from what I can tell it looks like a masterpiece

**Comment by steveoh** [comment]
have the related downloads been migrated to the share? Did the urls stay the same like we were thinking?

**Comment by rkelson** [comment]
yes, they were already on the public share and I tested the downloads

Rick Kelson
Utah AGRC
801.433.7891
gis.utah.gov

**Review by steveoh — APPROVED** [review]

**Pull request merged**